src/components/Drawer/Drawer.tsx

```diff
@@ -206,10 +207,26 @@ const Drawer = forwardRef<HTMLDivElement, DrawerProps>(
         <Portal>
           {hasBackdrop ? (
             <Overlay placement="right">
-              <DrawerBox {...drawerProps} />
+              <FocusTrap
```

Comment:
I wonder, since the same configuration of the FocusTrap component is used on all places. If it make sense to extract it and put into `_internal` directory so we don't need to repeat the same code everywhere

Reply:
Fixed